tests/LoadsImages.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace Grommet\ImageResizer\Tests;
+
+/**
+ * Loads images trait
+ */
+trait LoadsImages
+{
+    protected function resourceDir(string $sub = ''): string
+    {
+        $dir = dirname(__FILE__) . DIRECTORY_SEPARATOR . 'resources' . DIRECTORY_SEPARATOR;
+        if ($sub) {
+            $dir .= $sub . DIRECTORY_SEPARATOR;
```

Comment:
Is it possible for the `$sub` variable to already end with the `DIRECTORY_SEPARATOR`? Might be good to have a check to just do something like this:
```suggestion
            $dir .=  rtrim($sub, DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR;
```